pkg/api/operators/apis/trial/v1alpha2/trial_types.go

```diff
@@ -51,22 +58,7 @@ type TrialStatus struct {
 	Conditions []TrialCondition `json:"conditions,omitempty"`
 
 	// Results of the Trial - objectives and other metrics values.
-	Observation Observation `json:"observation,omitempty"`
-}
-
-type ParameterAssignment struct {
-	Name  string `json:"name,omitempty"`
-	Value string `json:"value,omitempty"`
-}
-
-type Metric struct {
-	Name  string  `json:"name,omitempty"`
-	Value float64 `json:"value,omitempty"`
-}
-
-type Observation struct {
-	// Key-value pairs for metric names and values
-	Metrics []Metric `json:"metrics"`
+	Observation common.Observation `json:"observation,omitempty"`
```

Comment:
I think it is better to use `*common.Observation` since it is omitempty

Reply:
Changed.